xapi-forkexecd.opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "1.2"
+maintainer: "xen-api@lists.xen.org"
+authors: "xen-api@lists.xen.org"
+homepage: "https://github.com/xapi-project/"
+bug-reports: "https://github.com/xapi-project/forkexecd/issues"
+dev-repo: "git://github.com/xapi-project/forkexecd"
+tags: [ "org:xapi-project" ]
+
+build: [[ "jbuilder" "build" "-p" name "-j" jobs ]]
+build-test: ["jbuilder" "runtest" "-p" name]
```

Comment:
This also takes the number of jobs: `"-j" jobs` -  see https://jbuilder.readthedocs.io/en/latest/usage.html#tests

Reply:
There is only a test that need to run in a single process

Reply:
But maybe this is also for the compilation of the tests?

Reply:
Or only for the compilation of the tests? 🤔

Reply:
No idea, probably for the compilation, but it's not clear from the documentation. Anyways, the test is a single file as well, so multiple jobs would not make any difference.